lib/v2/591/list.js

```diff
@@ -0,0 +1,211 @@
+                    <td>${house.floor_str}</td>
+                </tr>
+                <tr>
+                    <td>社區</td>
+                    <td>${house.community}</td>
+                </tr>
+                <tr>
+                    <td>地點</td>
+                    <td>${house.location}</td>
+                </tr>
+                <tr>
+                    <td>更新時間</td>
+                    <td>${house.refresh_time}</td>
+                </tr>
+                <tr>
+                    <td>標籤</td>
+                    <td>${tags}</td>
+                </tr>
+            </table>
+
+            <p>更多資訊請見 <a href="${itemUrl}">591 租屋</a></p>
+
+            <br />
+
+            <h3>更多圖片</h3>
+
+            <br />
+
+            <div id="more-pictures">
+                ${photo_list
+                    .slice(1)
+                    .map((photo) => `<img src="${photo}" style="margin-bottom: 20px;">`)
+                    .join('')}
+            </div>
+
+            <div id="hidden-meta" style="display: none;">
+                <div id="post-id">${post_id}</div>
+                <div id="item-url">${itemUrl}</div>
+                <div id="item-title">${house.title}</div>
+                <div id="item-price">${price}</div>
+                <div id="item-price-unit">${price_unit}</div>
+                <div id="item-kind-name">${house.kind_name}</div>
+                <div id="item-area">${house.area}</div>
+                <div id="item-floor-str">${house.floor_str}</div>
+                <div id="item-community">${house.community}</div>
+                <div id="item-location">${house.location}</div>
+                <div id="item-refresh-time">${house.refresh_time}</div>
+                <div id="item-photo-list">${photo_list.join(',')}</div>
+                <div id="item-rent-tag">${rent_tag.map((tag) => `<span>${tag.name}</span>`).join('')}</div>
+                <div id="item-room-str">${house.room_str}</div>
+                <div id="item-section-name">${house.section_name}</div>
+                <div id="item-street-name">${house.street_name}</div>
+                <div id="item-role-name">${house.role_name}</div>
+                <div id="item-contact">${house.contact}</div>
+                <div id="item-yesterday-hit">${house.yesterday_hit}</div>
+                <div id="item-is-vip">${house.is_vip}</div>
+                <div id="item-is-combine">${house.is_combine}</div>
+                <div id="item-hurry">${house.hurry}</div>
+                <div id="item-is-socail">${house.is_socail}</div>
+                <div id="item-surrounding">${house.surrounding}</div>
+                <div id="item-discount-price-str">${house.discount_price_str}</div>
+                <div id="item-cases-id">${house.cases_id}</div>
+                <div id="item-is-video">${house.is_video}</div>
+                <div id="item-preferred">${house.preferred}</div>
+                <div id="item-cid">${house.cid}</div>
+            </div>`;
```

Comment:
Please [use art-template](https://docs.rsshub.app/joinus/script-standard.html#template)
docs: https://aui.github.io/art-template/docs/syntax.html

Reply:
fixed in f607581